modules/virt-template-fields-web.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * virt/vm_templates/virt-creating-vm-template.adoc
+
+:_content-type: PROCEDURE
+[id="virt-template-fields-web_{context}"]
+= Custom template fields
+
+The following table outlines the editable fields when creating a new template in the {VirtProductName} web console.
+
+|===
+|Name |Description
+
+|Details
+a|
+* Labels
+* Annotations
+* Display name
+* Description
+* Workload profile
+* CPU/Memory
+* Boot mode
+* GPU devices
+* Host devices
+
+|YAML
+|To view the VM custom resource (CR).
+
+|Scheduling
+a|
+* Node selector
+* Tolerations
+* Affinity Rules
```

Comment:
should "Rules" be capitalized? It might be an error in the UI if it's the only thing in title case